go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/xeipuuv/gojsonschema v1.2.0
 	gopkg.in/AlecAivazis/survey.v1 v1.8.7
 	gopkg.in/yaml.v2 v2.2.4
+	gotest.tools v2.2.0+incompatible
```

Comment:
I'm pretty sure this line shouldn't be here, I bet you meant to use testify assert instead somewhere and instead the IDE pulled in this. This happens to me all the time now with go mods, I usually can just remove this line and commit the file and everything is fine.